cpp/cmake/slice2cpp.cmake

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) ZeroC, Inc.
+
+find_program(Ice_SLICE2CPP_EXECUTABLE slice2cpp
+  HINTS ${PACKAGE_PREFIX_DIR}
+  PATH_SUFFIXES bin ../../tools
```

Comment:
I think this relative path is correct for the nuget (CI will tell us). Would be nice to find a better way for this.